test/functional/client_side_encryption_tests.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const path = require('path');
+const TestRunnerContext = require('./runner').TestRunnerContext;
+const gatherTestSuites = require('./runner').gatherTestSuites;
+const generateTopologyTests = require('./runner').generateTopologyTests;
+
+const missingAwsConfiguration =
+  process.env.AWS_ACCESS_KEY_ID == null || process.env.AWS_SECRET_ACCESS_KEY == null;
+
+describe('Client Side Encryption', function() {
+  const testContext = new TestRunnerContext();
+  const testSuites = gatherTestSuites(path.join(__dirname, 'spec', 'client-side-encryption'));
+  after(() => testContext.teardown());
```

Comment:
q: can't recall, will the `before` and `after` scripts run if there are no tests?